Bullseye/app/src/main/AndroidManifest.xml

```diff
@@ -16,13 +16,13 @@
         <activity
             android:name=".AboutActivity"
             android:parentActivityName=".MainActivity"
-            android:screenOrientation="landscape"
+
             android:theme="@style/Theme.Bullseye"
             android:exported="false" />
         <activity
             android:name=".MainActivity"
             android:exported="true"
-            android:screenOrientation="landscape"

```

Comment:
To make the portrait view show in portrait mode, this line will then have to be removed after adding the layout.